content/chat/channels.textile

```diff
@@ -0,0 +1,105 @@
+---
+title: Channels and Messages
+meta_description: "Channels bring forward the capability to provide real-time communication between multiple people."
+product: chat
+languages:
+  - javascript
+---
+
+Channels are the core functionality for any chat application or any application with chat features. Channels enable you to separate chat channels and their messages into categories. For example, a single channel for direct messages between two people, or a channel for a specific live streaming event with many chat participants.
+
+Clients can subscribe to channels and publish messages using the realtime interface of an Ably SDK. The REST interface only allows clients to publish messages.
+
+<img src="/images/products/chat/chat.svg" alt="Example of a chat message sent by a user called Claire Jones, with reactions added to the message and a reply from Siti Tengku.">
+
+h2(#namespaces). Namespaces
+
+Group and organise the chat channels by using namespaces, which are based on a prefix that is included as part of the channel name. A colon (:) is used to delimit a channel namespace, where the first segment up until the first colon defines the namespace, any segments after this is defined by you. If a channel name does not contain a colon, the namespace is the entire channel name.
```

Comment:
Everything below just states 'channels' - so perhaps we either just state 'chats' or stick with just 'channels'.

Reply:
I've gone with consistency, removed the `chats` part. https://github.com/ably/docs/pull/2090/commits/a76a50801f6dc4d2fe2a79ff1c8713a80c8e3307